sycl/doc/extensions/supported/sycl_ext_oneapi_backend_level_zero.md

```diff
@@ -389,6 +412,67 @@ Construct a SYCL buffer instance from a pointer to a Level Zero memory allocatio
 description above for semantics and restrictions.
 The additional <code>AvailableEvent</code> argument must be a valid SYCL event. The instance of the SYCL buffer class template being constructed must wait for the SYCL event parameter to signal that the memory native handle is ready to be used.
 </tr>
+
+<tr>
+<td>
+
+``` C++
+make_image(

```

Comment:
I think you should show the template parameters for the `make_image` functions.  I know we don't shows them for the others, but it's less critical for those functions because the template parameters are specified in the SYCL 2020 spec.  However, SYCL 2020 doesn't have `make_image`, so there is currently no place where those template parameters are specified.

If you wanted to be consistent, we could add the template parameters for all these `make_OBJECT` functions.